api/src/passage/management/commands/passage_zone_zwaar_verkeer.py

```diff
@@ -0,0 +1,172 @@
+import datetime
+import logging
+from datetime import date, timedelta
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        # Named (optional) argument
+        parser.add_argument(
+            '--from-date',
+            type=date,
+            help='Run the aggregations from this date',
+        )
+
+    def _get_delete_query(self, run_date):
+        return f""" 
+        DELETE FROM passage_zonezwaarverkeer
+        WHERE passage_at_year = {run_date.year}
+        AND passage_at_month = {run_date.month}
+        AND passage_at_day = {run_date.day}
+        ;
+        """
+
+    def _get_aggregation_query(self, run_date):
+        return f"""
+        INSERT INTO passage_zonezwaarverkeer (
+            passage_at_timestamp,
+            passage_at_date,
+            passage_at_year,
+            passage_at_month,
+            passage_at_day,
+            passage_at_week,
+            passage_at_day_of_week,
+            passage_at_hour,
+            order_kaart,
+            order_naam,
+            cordon,
+            richting,
+            location,
+            geom,
+            azimuth,
+            kenteken_land,
+            voertuig_soort,
+            inrichting,
+            voertuig_klasse_toegestaan_gewicht,
+            intensiteit
+        )
+        
+        -- query voor zone zwaar verkeer
+        SELECT
+        date_trunc('hour', p.passage_at) AS timestamp,
+        date(p.passage_at) AS date,
+        extract(YEAR FROM p.passage_at)::int AS year,
+        extract(MONTH FROM p.passage_at)::int AS month,
+        extract(DAY FROM p.passage_at)::int AS day,
+        extract(WEEK FROM p.passage_at)::int AS week,
+        CASE
+            WHEN extract(DOW FROM p.passage_at)::int = 0 then '7 zondag'
+            WHEN extract(DOW FROM p.passage_at)::int = 1 then '1 maandag'
+            WHEN extract(DOW FROM p.passage_at)::int = 2 then '2 dinsdag'
+            WHEN extract(DOW FROM p.passage_at)::int = 3 then '3 woendsag'
+            WHEN extract(DOW FROM p.passage_at)::int = 4 then '4 donderdag'
+            WHEN extract(DOW FROM p.passage_at)::int = 5 then '5 vrijdag'
+            WHEN extract(DOW FROM p.passage_at)::int = 6 then '6 zaterdag'
+        ELSE 'onbekend ' END AS dow,
+        extract(HOUR FROM p.passage_at)::int AS hour,
+        --  blok camera informatie
+        h.order_kaart,
+        h.order_naam,
+        h.cordon,
+        h.richting,
+        h.location,
+        h.geom,
+        h.azimuth,
+        --  blok voertuig informatie
+        CASE WHEN p.kenteken_land = 'NL'then 'NL' ELSE 'buitenland' END AS kenteken_land,
+        p.voertuig_soort AS voertuig_soort,
+        CASE 	WHEN p.voertuig_soort = 'Personenauto' then 'Personenauto' ELSE inrichting END AS inrichting,
```

Comment:
het veld inrichting moet zijn "inrichting" of "Personenauto"? Klinkt niet logisch..?

Reply:
nee het is de tekst "Personenauto" als dat het voertuig soort is, en anders de waarde van `inrichting`